lib/linguist/languages.yml

```diff
@@ -1471,11 +1471,13 @@ Frege:
   ace_mode: haskell
   language_id: 116
 G-code:
-  type: data
+  type: programming
+  color: "#ff4d6b"
   extensions:
   - ".g"
   - ".gco"
   - ".gcode"
+  - ".cnc"
```

Comment:
For other reviewers: using Harvester, I counted 341 repositories by 240 users over a total of 2389 files checked.